### PR TITLE
Add command az containerapp show-custom-domain-verification-id

### PR DESCRIPTION
**Pull request by Seris370**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 1]
```diff
@@ -1623,6 +1623,12 @@ def test_containerapp_create_with_vnet_yaml(self, resource_group):
 
 
 class ContainerappOtherPropertyTests(ScenarioTest):
+    # @live_only()
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:1626]: please remove this line if useless.

**Comment by Greedygre** [comment]
LGTM.
@Juliehzl Could please help to review this PR.

**Review by Greedygre — APPROVED** [review]

**Review by Greedygre — APPROVED** [review]

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 2]
```diff
@@ -1623,6 +1623,11 @@ def test_containerapp_create_with_vnet_yaml(self, resource_group):
 
 
 class ContainerappOtherPropertyTests(ScenarioTest):
+    def test_containerapp_get_customdomainverificationid(self):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        self.cmd(f'containerapp show-custom-domain-verification-id', expect_failure=False)
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:1629]: could you also add command to use it bind domain?

File: src/containerapp/azext_containerapp/_help.py [hunk 3]
```diff
@@ -201,6 +201,11 @@
           az containerapp logs show -n MyContainerapp -g MyResourceGroup --replica MyReplica --revision MyRevision --container MyContainer
 """
 
+helps['containerapp show-custom-domain-verification-id'] = """
+    type: command
+    short-summary: Show the verification id for binding custom domains.
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/_help.py:206]: add examples

File: src/containerapp/azext_containerapp/_clients.py [hunk 4]
```diff
@@ -1517,6 +1517,15 @@ def get(cls, cmd, resource_group_name, container_app_name, auth_config_name):
 class ContainerAppPreviewClient(ContainerAppClient):
     api_version = PREVIEW_API_VERSION
 
+    @classmethod
+    def show_custom_domain_verification_id(cls, cmd):
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/_clients.py:1521]: it belongs to containerappClient or env client?

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py (hunk 2 above)
Comment (Seris370) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py]: Done.

File: src/containerapp/azext_containerapp/_help.py (hunk 3 above)
Comment (Seris370) [reply src/containerapp/azext_containerapp/_help.py]: Added

File: src/containerapp/azext_containerapp/_clients.py (hunk 4 above)
Comment (Seris370) [reply src/containerapp/azext_containerapp/_clients.py:1529]: Added SubscriptionClient

File: src/containerapp/azext_containerapp/_help.py [hunk 5]
```diff
@@ -201,6 +201,15 @@
           az containerapp logs show -n MyContainerapp -g MyResourceGroup --replica MyReplica --revision MyRevision --container MyContainer
 """
 
+helps['containerapp show-custom-domain-verification-id'] = """
+    type: command
+    short-summary: Show the verification id for binding app or environment custom domains
+    examples:
+    - name: Get the verification id, which needs to be added as a TXT record for app custom domain to verify domain ownership
+      text: |
+          containerapp show-custom-domain-verification-id
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/_help.py:210]:
```suggestion
          az containerapp show-custom-domain-verification-id
```

File: src/containerapp/azext_containerapp/_help.py [hunk 6]
```diff
@@ -201,6 +201,15 @@
           az containerapp logs show -n MyContainerapp -g MyResourceGroup --replica MyReplica --revision MyRevision --container MyContainer
 """
 
+helps['containerapp show-custom-domain-verification-id'] = """
+    type: command
+    short-summary: Show the verification id for binding app or environment custom domains
+    examples:
+    - name: Get the verification id, which needs to be added as a TXT record for app custom domain to verify domain ownership
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/_help.py:208]: app custom domain? should be env or app?

**Review by Juliehzl — APPROVED** [review]

**Comment by Greedygre** [comment]
Hi @zhoxing-ms 
This PR has been review by me and zunli, could you help to review and merge? Thanks.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**